### ISSUE TITLE
Moves after #748 return Not Found 

### ISSUE THREAD

**Issue by IlanDeVinci** [PokeAPI/pokeapi#1109]
Steps to Reproduce:
https://pokeapi.co/api/v2/move/749

All moves with a higher number than or equal to 749 will return "Not Found" instead of the data.
1.
2.

**Comment by Naramsim** [comment]
Hi! It seems a problem with the `ditto` utility we use. I'm looking into it! Thanks for the issue!

**Comment by Naramsim** [comment]
Hi! I regenerated the data.Now Cloudflare caches 404 for 5 days, so I custom purged some entries, but most probably you'll have to wait 5 days for every move to be available

**Comment by IlanDeVinci** [comment]
Thank you for the quick response and fix !

**Comment by GreatNovaDragon** [comment]
this shifted the problem only moved up to 775, clangurous-soul

**Comment by monktana** [comment]
Hello, quick question:
Currently, i get a Not Found response for certain moves, depending on whether the url ends with a "/" or not.

Example:
https://pokeapi.co/api/v2/move/883 --> response is Not Found
https://pokeapi.co/api/v2/move/883/ --> response is the move data (snowscape)

If this needs a separate issue, please let me know.

**Comment by Naramsim** [comment]
It's the same issue, I probably purged the non slash version or viceversa
